Sources/EdgeBridge.swift

```diff
@@ -83,28 +83,28 @@ public class EdgeBridge: NSObject, Extension {
             return
         }
 
-        dispatchTrackRequest(data: consequenceDetail, timestamp: event.timestamp)
+        dispatchTrackRequest(data: consequenceDetail, parentEvent: event)
     }
 
     /// Helper to create and dispatch an experience event.
     /// - Parameters:
     ///   - data: dictionary containing free-form data to send to Edge Network
-    ///   - timestamp: timestamp of Event
-    private func dispatchTrackRequest(data: [String: Any], timestamp: Date) {
+    ///   - parentEvent: the triggering parent event
```

Comment:
can we update description here, e.g. 
parentEvent: the triggering parent event used for event chaining; its timestamp is set as xdm.timestamp